### DISCUSSION TITLE
field for oven in recipe

### DISCUSSION THREAD

**Discussion by TWART016** [TandoorRecipes/recipes#1370] [Ideas]
Sometimes the instructions say how long the dish must be in the oven. For this, the oven must be preheated. This is usually written in the body text of the dishes, which is of course very difficult to read out. 
Does it make sense to create a field for this in the description under keywords?

Content: Upper- lower heat 200°

**Comment by vabene1111** [comment]
i think the most appropriate soultion for this (and the way i handle it as well) is to create a step specifically for pre heating the oven. This way you can arrange it within the recipe and you can even add a time to it so you know how long to wait before or after performing other tasks

**Reply by vabene1111** [reply]
also i usually mark that bold so i can see it easier, i wanted to add some custom icons at some point for the oven settings, so its even easier to read. But utensils is something we are planning to add and i could imagine utensil preparation being a thing there as well, so that could work basically as you intend with the extra field

**Reply by TWART016** [reply]
That sounds good. 
Possibly one could connect this with a timer. I think there is still a feature request for that.

**Comment by smilerz** [comment]
An ingredient header would work as well and would appear in the ingredient summary. .

**Reply by TWART016** [reply]
You are right. Everything is bold that everything is written in bold. I find that not ideal.

And a separator line would be great in the ingredients list
